README.md

```diff
@@ -694,7 +695,9 @@ Usage: `nerdctl logs [OPTIONS] CONTAINER`
 Flags:
 - :whale: `--f, --follow`: Follow log output
 - :whale: `--since`: Show logs since timestamp (e.g. 2013-01-02T13:23:37Z) or relative (e.g. 42m for 42 minutes)
+  - for journald log driver logs since relative should be in this format `42 minutes ago`
```

Comment:
Can we just use the same format?
If difficult this flag can be implemented in a separate PR.